tests/test_photon_ops.py

```diff
@@ -162,8 +179,8 @@ def test_rubin_optics() -> None:
     rubin_optics.applyTo(photon_array, local_wcs=local_wcs, rng=create_test_rng())
     np.testing.assert_array_equal(photon_array.pupil_u, u)
     np.testing.assert_array_equal(photon_array.pupil_v, v)
-    expected_x_pic_center = 564.5
-    expected_y_pic_center = -1431.4
+    expected_x_pic_center = -989.5971378245167
+    expected_y_pic_center = -3840.3512012842157
```

Comment:
Why are these (and subsequent tests) changing so much?  This isn't close.  

Regression tests are supposed to make sure we don't change the functionality of existing code.  I know the implementation of these photon ops changed, but I think we want the tests to remain the same or at least very very close.  What happened here?

Reply:
This is still a big issue. I'm 99% convinced the error is in the original test itself. My current best guess is that with the old code it may not have been internally self-consistent around the WCS used, while the new `XyToV` should enforce self-consistency. But it still needs to be looked at.